configure.ac

```diff
@@ -1016,6 +1016,43 @@ if test x$enable_monotouch = xyes; then
 	AC_DEFINE(ENABLE_MONOTOUCH, 1, [Enable runtime support for Monotouch (Xamarin.iOS and Xamarin.Mac)])
 fi
 
+AC_ARG_ENABLE(cplusplus, [  --enable-cplusplus   compile some code as C++])
+
+if test "x$enable_cplusplus" = "xyes"; then
```

Comment:
cxx is used everywhere else than here.

Reply:
You mean, you suggest?or?mandate:
configure/autogen -enable-cxx
CXX_ADD_CFLAGS
CXX_REMOVE_CFLAGS
CI_TAGS=cxx
etc.?
Otherwise please elaborate.
Consistency is arguable, but brevity is not, so ok.